.docker/Dockerfile

```diff
@@ -114,6 +90,4 @@ WORKDIR /srv/jekyll
 EXPOSE 35729
 EXPOSE 4000
 
-CMD ["jekyll", "serve", "--livereload", "-i", "--force_polling", "--watch", "--host", "0.0.0.0"]
```

Comment:
I'd like to have a default `CMD` even if that ends up being `CMD ["jekyll", "help"]`
I see there is a lot of magic happening with command checking in the `entrypoint` file.
As it is now, the default command is a bit obscured.